web/writing.html

```diff
@@ -374,13 +315,100 @@ <h5 class="portlet__head">Webhook</h5>
         <button class="save-button btn btn-primary submit_btn" style="margin-bottom: 16px;">Save</button>
 
       </div>
-      <div class="col-md-3">
-        <button type="button" class="btn btn-outline-primary btn-sm" style="margin-top: 68px;"
-          id="clearLogBtn">清空日志</button>
-        <p class="logs font-weight-light text-break" style="margin-top: 20px;font-size: 13px;" id="logs"></p>
+      <div class="logs-panel col-md-6 offset-md-3" style="visibility: hidden;">
+        <textarea class="logs form-control" id="logs" disabled>
+        </textarea>
```

Comment:
```suggestion
        <textarea class="logs form-control" id="logs" 
          readonly></textarea>
```

`textarea` 会将开始标签和结束标签[之间的内容设为值](https://stackoverflow.com/questions/6007219/how-to-add-default-value-for-html-textarea)，所以换行会导致 `logsDom.value` 默认不为空而是 `        `。

另外，使用 [`readonly`](https://developer.mozilla.org/docs/Web/HTML/Attributes/readonly) 代替 `disabled` 是不是更好些？

Reply:
可以用readonly，没了解过这个用法，其实不影响日志的显示效果